### PR TITLE
New catalog layout

### PR DESCRIPTION
**Pull request by quintesse**

### CODE REVIEW

**Comment by quintesse** [comment]
Test should fail because the catalog hasn't been switched over yet

File: addons/launcher-addon/src/test/resources/fetch-booster-bundles.sh [hunk 1]
```diff
@@ -6,15 +6,11 @@ mkdir -p ${TEST_DIR} > /dev/null 2>&1
 BOOSTER_TMP_DIR="${TEST_DIR}/booster-catalog"
 git clone git@github.com:fabric8-launcher/launcher-booster-catalog.git ${BOOSTER_TMP_DIR}
 cd ${BOOSTER_TMP_DIR}
-for branch in `git branch -a | grep remotes | grep -v HEAD | grep -v master `; do
-   git branch --track ${branch#remotes/origin/} $branch
```
Comment (gastaldi) [inline addons/launcher-addon/src/test/resources/fetch-booster-bundles.sh]: Won't this command change to the correct git branch?

File: addons/legacy-osio-addon/src/main/java/io/fabric8/forge/generator/quickstart/ValidBoosters.java [hunk 2]
```diff
@@ -1,68 +0,0 @@
...
-import java.util.HashSet;
-import java.util.Set;
-
-import io.fabric8.launcher.booster.catalog.Booster;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-/**
- */
-public class ValidBoosters {
```
Comment (gastaldi) [inline addons/legacy-osio-addon/src/main/java/io/fabric8/forge/generator/quickstart/ValidBoosters.java]: YES! Goodbye headache! :+1:

File: addons/launcher-addon/src/test/resources/fetch-booster-bundles.sh (hunk 1 above)
Comment (quintesse) [reply addons/launcher-addon/src/test/resources/fetch-booster-bundles.sh]: There is no "correct" git branch anymore, we only have "master" nowadays for the catalog, don't we?

Comment (gastaldi) [reply addons/launcher-addon/src/test/resources/fetch-booster-bundles.sh]: Ah, I thought those were for the boosters. In this case, nvm

**Pull request merged**